grakn-core/src/main/java/ai/grakn/concept/Thing.java

```diff
@@ -80,6 +80,14 @@
     Thing resource(Resource resource);
 
     /**
+     * Creates a relationship from this instance to the provided resource.
```

Comment:
For the sake of my future sanity please to the following:

```
Creates a {@link Relationship} from this {@link Thing} to the provided {@link Attribute}.
```

I know it's more tedious but it makes refactors 100x easier